### PR TITLE
Add Eli

### PR DESCRIPTION
**Pull request by KrisXV**

### CODE REVIEW

File: data/mods/ssb/conditions.ts [hunk 1]
```diff
@@ -123,6 +123,19 @@ export const Conditions: {[k: string]: ModdedConditionData & {innateName?: strin
...
+		noCopy: true,
+		onStart() {
+			this.add(`c:|${getName('Eli')}|any`);
+		},
+		onSwitchOut() {
+			this.add(`c:|${getName('Eli')}|ok bye`);
+		},
+		onFaint(pokemon) {
+			const foePokemon = (pokemon.side.foe.active[0].illusion || pokemon.side.foe.active[0]).name;
+			this.add(`c:|${getName('Eli')}|that wasn't very nice, ${foePokemon}.`);
```
Comment (HisuianZoroark) [inline data/mods/ssb/conditions.ts:136]:
```suggestion
			this.add(`c:|${getName('Eli')}|that wasn't very nice, ${pokemon.foes()[0].name}.`);
```

File: data/mods/ssb/moves.ts [hunk 2]
```diff
@@ -181,6 +181,7 @@ export const Moves: {[k: string]: ModdedMoveData} = {
 			return move.basePower;
 		},
 		category: "Physical",
+		shortDesc: "The person who coded this didn't write one.",
```
Comment (HisuianZoroark) [inline data/mods/ssb/moves.ts:184]:
```suggestion
		shortDesc: "If foe is switching out, 2x power. Doesn't KO.",
```

**Comment by HisuianZoroark** [comment]
I am concerned that things will need to be refactored in order to support this custom weather (does thunder not miss? How does weather ball work here?) but for all intents and purposes this should be good to go

**Review by HisuianZoroark — APPROVED** [review]

**Comment by KrisXV** [comment]
> 

we can handle this later

**Pull request merged**